setup.cfg

```diff
@@ -4,7 +4,7 @@
 # https://setuptools.pypa.io/en/latest/references/keywords.html
 
 [metadata]
-name = preset-cli
+name = preset-cli-a8c
```

Comment:
Was this intentional?

Reply:
Sorry, it was left during the tests. I've removed it.

Reply:
no worries, and thank you! 🙏